src/include/daos/object.h

```diff
@@ -121,7 +121,8 @@ typedef struct {
 struct daos_obj_md {
 	daos_obj_id_t		omd_id;
 	uint32_t		omd_ver;
-	uint32_t		omd_padding;
+	/* Performance domain affinity */
+	uint32_t		omd_pda;
```

Comment:
Do we need two value to differ rc and replica? eg, omd_ec_pda, omd_rp_pda as HLD suggested?

Reply:
I think one is fine, because here is to place obj and we already know its type (EC or replica).